lib/generators/decidim/install_generator.rb

```diff
@@ -110,8 +115,19 @@ def letter_opener_web
       def recreate_db
         soft_rails "db:environment:set", "db:drop"
         rails "db:create"
-        rails "db:migrate"
-        rails "db:seed" if options[:seed_db]
+
+        # In order to ensure that migrations don't eager load models with not
+        # yet fully populated schemas (which could break commands which load
+        # migrations and seeds in the same process, such as `rails db:migrate
+        # db:seed`), we make sure to run them in the same process if seeds are
+        # requested so that we can catch these situations earlier than end
+        # users.
+        if options[:seed_db]
```

Comment:
I actually splitted this some time ago because seeding was erroring when running in the same process as migrations 🤨

Reply:
Yes, now it's not. Maybe the solution is to never run that command but I find it handy to seed and migrate in one command... 😔